mlbench_core/controlflow/pytorch/controlflow.py

```diff
@@ -37,7 +37,7 @@ def _record_train_batch_stats(batch_idx, loss, output, target, metrics,
 
     # Compute metrics for one batch
     for metric in metrics:
-        metric_value = metric(output, target).item()
+        metric_value = metric(loss, output, target).item()
```

Comment:
Would this work with the metrics used for [CIFAR10](https://github.com/mlbench/mlbench-benchmarks/blob/896d3c4f31aa7205ad9fcedebd5dc46e3b8e36ed/pytorch/imagerecognition/openmpi-cifar10-resnet20-all-reduce-scaling/main.py#L81)  ? Versions make it okay for now but we need to keep in mind when updating versions